mod_modular_vanilla/hooks/entity/tactical/actor.nut

```diff
@@ -46,6 +46,14 @@
 });
 
 ::ModularVanilla.MH.hook("scripts/entity/tactical/actor", function (q) {
+	// MV: Added
+	// Part of the actor.MV_interrupt framework
```

Comment:
```suggestion
	// Part of the actor.MV_interrupt framework
	// Cause an 'Interruption' on this actor. That is a new standardized event, which will remove some temporary skills on the target. In Vanilla those interruptible skills are shieldwall, riposte and spearwall
```

I suggest we give more context here for modder, as this is the most public facing function of this framework